<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>POI_editor_ui</class>
 <widget class="QDialog" name="POI_editor_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">POI :</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Nom</string>
   </property>
   <widget class="QLineEdit" name="editName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>471</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Position</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Longitude Est</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="lon_deg">
       <property name="value">
        <number>-89</number>
       </property>
       <property name="suffix">
        <string> °</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>-359</number>
       </property>
       <property name="maximum">
        <number>359</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lon_min">
       <property name="value">
        <double>59.990000000000002</double>
       </property>
       <property name="suffix">
        <string> '</string>
       </property>
       <property name="maximum">
        <double>59.990000000000002</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>451</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Prochain cap</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editWph"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Latitude Nord</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="lat_deg">
       <property name="value">
        <number>-89</number>
       </property>
       <property name="suffix">
        <string> °</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>-89</number>
       </property>
       <property name="maximum">
        <number>89</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lat_min">
       <property name="value">
        <double>59.990000000000002</double>
       </property>
       <property name="suffix">
        <string> '</string>
       </property>
       <property name="maximum">
        <double>59.990000000000002</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>471</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Horadatage de passage</string>
   </property>
   <widget class="QCheckBox" name="chk_tstamp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Activer</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="editTStamp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>77</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btOk">
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btCancel">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btDelete">
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btCopy">
      <property name="text">
       <string>Copier</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btPaste">
      <property name="text">
       <string>Coller</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btSaveWP">
      <property name="text">
       <string>POI-&gt;WP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>lat_deg</tabstop>
  <tabstop>lat_min</tabstop>
  <tabstop>lon_deg</tabstop>
  <tabstop>lon_min</tabstop>
  <tabstop>editWph</tabstop>
  <tabstop>chk_tstamp</tabstop>
  <tabstop>editTStamp</tabstop>
  <tabstop>btOk</tabstop>
  <tabstop>btCancel</tabstop>
  <tabstop>btDelete</tabstop>
  <tabstop>btCopy</tabstop>
  <tabstop>btPaste</tabstop>
  <tabstop>btSaveWP</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btOk</sender>
   <signal>clicked()</signal>
   <receiver>POI_editor_ui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancel</sender>
   <signal>clicked()</signal>
   <receiver>POI_editor_ui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btDelete</sender>
   <signal>clicked()</signal>
   <receiver>POI_editor_ui</receiver>
   <slot>btDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCopy</sender>
   <signal>clicked()</signal>
   <receiver>POI_editor_ui</receiver>
   <slot>btCopyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btPaste</sender>
   <signal>clicked()</signal>
   <receiver>POI_editor_ui</receiver>
   <slot>btPasteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSaveWP</sender>
   <signal>clicked()</signal>
   <receiver>POI_editor_ui</receiver>
   <slot>btSaveWPClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_tstamp</sender>
   <signal>stateChanged(int)</signal>
   <receiver>POI_editor_ui</receiver>
   <slot>chkTStamp_chg(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>POI_editor_ui</receiver>
   <slot>nameHasChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btDeleteClicked()</slot>
  <slot>btPasteClicked()</slot>
  <slot>btCopyClicked()</slot>
  <slot>btSaveWPClicked()</slot>
  <slot>chkTStamp_chg(int)</slot>
  <slot>nameHasChanged(QString)</slot>
 </slots>
</ui>
